.github/workflows/fork-ci.yaml

```diff
@@ -74,34 +74,42 @@ jobs:
 
   update_network_operator_values:
     needs:
-    - determine_docker_registry_and_tag
-    - build_and_push_images
+      - determine_docker_registry_and_tag
+      - build_and_push_images
     runs-on: ubuntu-latest
     env:
       DOCKER_TAG: ${{ needs.determine_docker_registry_and_tag.outputs.docker_tag }}
       GH_TOKEN: ${{ secrets.GH_TOKEN_NVIDIA_CI_CD }}
     steps:
-    - uses: actions/checkout@v4
-      with:
-        token: ${{ secrets.GH_TOKEN_NVIDIA_CI_CD }}
-        repository: ${{ github.repository_owner }}/network-operator
-    - name: Create PR to update image tags in network-operator values
-      run: |
-        git config user.name  nvidia-ci-cd
-        git config user.email svc-cloud-orch-gh@nvidia.com
-
-        git checkout -b feature/update-sriov-tags-to-$DOCKER_TAG
-        sed -Ei "s/(version: )network-operator-.+/\\1$DOCKER_TAG/g" hack/release.yaml
-        make release-build
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GH_TOKEN_NVIDIA_CI_CD }}
+          repository: ${{ github.repository_owner }}/network-operator
+      - name: Check if beta build
+        run: echo "IS_BETA=$(echo $DOCKER_TAG | awk -F  '-v' '/1/ {print $2}' | grep 'beta'" >> $GITHUB_ENV
+      - name: Get branch name
+        run: |
+          if [ -z "${IS_BETA}" ]; then
+            echo "BRANCH_NAME=master" >> $GITHUB_ENV
+          else
+            echo "BRANCH_NAME="$DOCKER_TAG" >> $GITHUB_ENV
+          fi
```

Comment:
I believe this can be shortened to:
```yaml
    - name: Determine base branch
      run: |
        echo "BASE_BRANCH=${{ contains(env.DOCKER_TAG, 'beta') && 'main' || env.DOCKER_TAG }}" >> $GITHUB_ENV
```